onnxruntime/core/graph/conv_add_fusion.cc

```diff
@@ -22,6 +22,13 @@ Status ConvAddFusion::Apply(onnxruntime::Graph& graph, bool& modified) const {
       continue;
     }
 
+    assert(node.Op()->SinceVersion() == 1 && next_node.Op()->SinceVersion() == 7);
```

Comment:
Would it be cleaner pattern to have a common helper that checks the OpType + Domain and then also do the Deprecated check and a known version?

Reply:
Yeah, I am thinking about it.

Reply:
Add it to GraphTransformerManager::ApplyAll() method?

Reply:
No, these code is not generic for all transformers. I will create a help function for it.

Reply:
@tracysh A helper function is created. Please review it.

Reply:
Thanks, looks better to me.